modules/nf-core/freyja/boot/main.nf

```diff
@@ -0,0 +1,58 @@
+process FREYJA_BOOT {
+    tag "$meta.id"
+    label 'process_long'
+
+    conda "bioconda::freyja=1.3.12"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/freyja:1.3.12--pyhdfd78af_0':
+        'quay.io/biocontainers/freyja:1.3.12--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta),  path(variants)
+    tuple val(meta2), path(depths)
+    val repeats
+    tuple val(meta3), path (barcodes)
+    tuple val(meta4), path (lineages_meta)
```

Comment:
Wondering why we need `meta` for all of these inputs @nvnieuwk ? Are `depths`, `barcodes` and `lineages` tied to individual samples or will they be the same across all samples fed to the pipeline @Joon-Klaps ? If the latter, then I think we can remove the `meta`. Just means unnecessary channel manipulations will be required.

Reply:
Same comment for other modules added in this PR.

Reply:
That was something we agreed on with the maintainers 🤷. I agree it doesn't make sense in some cases but guidelines are guidelines? 😁